variables.tf

```diff
@@ -97,6 +97,12 @@ variable "en_source_description" {
   description = "Optional description to give for the Event Notifications integration source. Only used if a value is passed for `en_instance_crn`."
 }
 
+variable "skip_en_s2s_auth_policy" {
+  type        = bool
+  default     = false
```

Comment:
If you also add `nullable = false` to these types of inputs, it will use the default value instead of `null` which helps avoid issues in your code (if you are not directly checking for null value). Very helpful for booleans like this.